core-test/src/test/scala/org/bitcoins/core/util/BitcoinScriptUtilTest.scala

```diff
@@ -32,6 +33,38 @@ class BitcoinScriptUtilTest extends FlatSpec with MustMatchers {
       .isEmpty must be(true)
   }
 
+  it must "filter out all but the data tokens" in {
+    val (p2pkhScript, _) = ScriptGenerators.p2pkhScriptPubKey.sample.get
```

Comment:
Please use property based testing correctly by segregating these test cases and using a `forAll()`